alpha/apps/kaltura/lib/storage/kFileSyncUtils.class.php

```diff
@@ -252,7 +279,46 @@ public static function file_put_contents ( FileSyncKey $key , $content , $strict
 		kFile::filePutContents($fullPath , $content);
 		self::setPermissions($fullPath);
 		self::createSyncFileForKey($rootPath, $filePath,  $key , $strict , !is_null($res), false, md5($content), kPathManager::getStorageProfileIdForKey($key));
-		self::encryptByFileSyncKey($key, kPathManager::getStorageProfileIdForKey($key));
+		$encryptedContent = self::encryptByFileSyncKey($key, kPathManager::getStorageProfileIdForKey($key));
+		$contentToPut = $encryptedContent ? $encryptedContent : $content;
```

Comment:
```suggestion
		$contentToPut = gzcompress($contentToPut) ?? $content;
```
Shorter ternary operator, which also won't fail on NULL value
Can use gzcompress and pass the result